webapp/WEB-INF/templates/skin/plugins/appointment/appointment_form_list.html

```diff
@@ -2,8 +2,12 @@
  	<div class="col-xs-12">
 		<div class="container">
 			<@messages infos=infos errors=errors />		
-			<h3><#if title??>${title}<#else>#i18n{appointment.xpage.formList.labelFormList}</#if></h3>
-			
+			<#if portlet_name?has_content>
+				<h3>${portlet_name}</h3>
+			<#elseif !portlet_name??>

```

Comment:
<#else> should be enough ?

Reply:
Hi, this has been updated:
1. When the template is used in a Portlet, to display a List of Appointment Forms:
1.1. Then display the custom Portlet's name, if it has content
1.2. Don't display anything if it is empty

3. In every other situation we display the default name from the resources